### PR TITLE
[MM-61132] Fix example for mmctl permissions role show command

### PR DESCRIPTION
**Pull request by hanzei**
#### Ticket Link
https://mattermost.atlassian.net/browse/MM-61132

#### Release Note
```release-note
Fix example for mmctl permissions role show command
```

### CODE REVIEW

File: server/cmd/mmctl/commands/permissions.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ var RemovePermissionsCmd = &cobra.Command{
 
 var ShowRoleCmd = &cobra.Command{
 	Use:        "show <role_name>",
-	Deprecated: "please use \"role show\" instead",
+	Deprecated: "please use \"mmctl permissions role show\" instead",
```
Comment (hanzei) [inline server/cmd/mmctl/commands/permissions.go:45]: I found the deprecation warning a bit hard to read. 

**New**
```bash
$ mmctl permissions show system_user

Command "show" is deprecated, please use "mmctl permissions role show" instead
...
```

**Old**
```bash
$ mmctl permissions show system_user

Command "show" is deprecated, please use "role show" instead
...
```

Comment (hanzei) [reply server/cmd/mmctl/commands/permissions.go:45]: Please let me know what you think. If you like the change, I will applied it to the other deprecated commands.

**Review by mgdelacroix — APPROVED** [review]
LGTM! :+1:

File: server/cmd/mmctl/commands/permissions.go (hunk 1 above)
Comment (hanzei) [reply server/cmd/mmctl/commands/permissions.go:45]: @mgdelacroix What do you think?

Comment (mgdelacroix) [reply server/cmd/mmctl/commands/permissions.go:45]: @hanzei when I saw it at first I was about to disagree, as I think we should provide instructions as simple as possible, but then I saw your examples with the command just atop, and I think it's easy enough to understand what should be changed and how.

Honestly I still think the first approach causes a bit less confusion than the second (most of the time when you're changing the command and don't want to think too much), but nothing that can't be solved going back, reading with a bit more focus and trying again. I'm just 1/5, so I'm perfectly happy with the change if you feel strongly about it.

Comment (hanzei) [reply server/cmd/mmctl/commands/permissions.go:45]: Should we apply the same change to the examples? Take, e.g. the `mmctl channel rename` command:

Screenshot from 2024-10-17 13-38-37

The Usage does include `mmctl` as a prefix, but the examples don't. The help text could be clearer by prefixing the root command.

Comment (hanzei) [reply server/cmd/mmctl/commands/permissions.go:45]: 0/5 if we want to do that as part of this PR. This conversation is extending beyond the original scope of the PR.

**Review by isacikgoz — APPROVED** [review]

**Pull request merged**